android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -726,6 +726,45 @@ public void peerConnectionAddICECandidate(ReadableMap candidateMap, final int id
         callback.invoke(result);
         Log.d(TAG, "peerConnectionAddICECandidate() end");
     }
+    private WritableArray convertWebRTCStats(StatsReport[] wrtcReports) {
```

Comment:
I believe we should add javadocs to the methods and fields we add. I'm open to discussion.